web/styles/recent_view.css

```diff
@@ -167,7 +167,8 @@
         border: 1px solid var(--color-border-zulip-button);
         border-radius: 40px;
         margin: 0 5px 10px 0;
```

Comment:
Should these other fields be converted too?

Reply:
They didn't stand out as much to me as looking weird, but it probably wouldn't hurt? Curious what Aman thinks.

Reply:
Yeah, margin between elements is one of the important ones to convert.

Reply:
Converted!

14px

<img width="927" alt="image" src="https://github.com/user-attachments/assets/c39d37f9-ff4a-42e0-8f4c-321d25957f85" />

16px

<img width="1032" alt="image" src="https://github.com/user-attachments/assets/870db6fa-578b-41e0-a2cb-c8a17a8d8735" />

18px

<img width="1186" alt="image" src="https://github.com/user-attachments/assets/2e05311e-3e7c-4749-9588-823a007ac906" />

20px

<img width="1369" alt="image" src="https://github.com/user-attachments/assets/40fe18e7-fa6d-4209-aec0-3350a97f9aec" />


---

It seems like the filter search bar should extend further right to align with the white elements above and below it, but I think that's a separate issue (that's also on main, so not related to info density?)